CMakeLists.txt

```diff
@@ -27,7 +30,7 @@ project(BitcoinCore
   VERSION ${CLIENT_VERSION_MAJOR}.${CLIENT_VERSION_MINOR}.${CLIENT_VERSION_BUILD}
   DESCRIPTION "Bitcoin client software"
   HOMEPAGE_URL "https://bitcoincore.org/"
-  LANGUAGES CXX
+  LANGUAGES NONE
```

Comment:
Is the reason for switching to None to stop cmake from doing a compiler/language check at this point which will fail on a missing install_name_tool?

The check will occur later with call to `enable_language`?